pkg/deploy/deploy.go

```diff
@@ -398,6 +405,61 @@ func createJobFromCronjob(k8sClient dynamic.Interface, res *unstructured.Unstruc
 	return jobCreated.GetName(), nil
 }
 
+func withAwaitableJob(timeout time.Duration, apply applyFunction) applyFunction {
```

Comment:
An interesting feature could be await all the resources deployed to see if all is correctly deployed. Can we make it more generic?

Reply:
:+1: